src/Apps/Artwork/Components/TrustSignals/VerifiedSeller.tsx

```diff
@@ -0,0 +1,42 @@
+import { VerifiedIcon } from "@artsy/palette"
+import { VerifiedSeller_artwork } from "__generated__/VerifiedSeller_artwork.graphql"
+import React from "react"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "react-relay"
+import { TrustSignal, TrustSignalProps } from "./TrustSignal"
+
+interface VerifiedSellerProps
+  extends Omit<TrustSignalProps, "Icon" | "label" | "description"> {
+  artwork: VerifiedSeller_artwork
+}
+
+export const VerifiedSeller: React.FC<VerifiedSellerProps> = ({
+  artwork,
+  ...other
+}) => {
+  return (
+    artwork.partner &&
+    artwork.partner.isVerifiedSeller && (
```

Comment:
We should probably also check for `isBiddable` here! Just in the case of a gallery auction.

Reply:
🙌  @sweir27. Added it in https://github.com/artsy/reaction/pull/2896/commits/3bac79ea1e8c6d76751b2259cdadf8f67e5bd111. Just wanted to double check I got that right.